diff --git a/usr/palm/frameworks/mojo/builtins/palmInitFramework200_47.js b/usr/palm/frameworks/mojo/builtins/palmInitFramework200_47.js
index 6beedfb..11949ef 100644
--- a/usr/palm/frameworks/mojo/builtins/palmInitFramework200_47.js
+++ b/usr/palm/frameworks/mojo/builtins/palmInitFramework200_47.js
@@ -36592,6 +36592,7 @@ autoCorrect:function(delim){
 
 /* oskb */
 isAutoCorrectDelim:function(letter){
+return false;
 var delims = " ,.'\";:?!-";
 
 return (letter && (delims.indexOf(letter) >= 0));
@@ -36616,6 +36617,7 @@ keyCode=chr[this.mode]&&chr[this.mode].keyCode || chr['normal'].keyCode;
 
 selection=this.controller.window.getSelection();
 
+/* eg:ac
 if (this.isAutoCorrectDelim(letter)) {
 	this.correctedText = this.target.value + letter;
 	this.target.value = this.autoCorrect(letter);
@@ -36623,6 +36625,7 @@ if (this.isAutoCorrectDelim(letter)) {
 		this.correctedText = false;
 	}
 }
+*/
 
 if (keyCode === 1 && !this.scrollEnabled) {
   this.dragEnabled = true;
@@ -36686,9 +36689,11 @@ this.controller.document.execCommand("insertText",true,letter);
 this.target.mojo.insertStringAtCursor(letter);
 }
 
+/* eg:ac
 if (!this.isAutoCorrectDelim(letter)) {
 	this.correctedText = false;
 }
+*/
 
 if (this.isSpecialChar(keyCode)) {
   Mojo.Event.send(this.target,'keydown',{keyCode:keyCode});
